.github/workflows/playwright.yml

```
@@ -0,0 +1,37 @@
+name: Playwright Tests
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
```

I see several places are depending on the `CI` env in the playwright config file, so we probably want to set it here.
```suggestion
    runs-on: ubuntu-latest
    env:
       CI: true
```